engine/src/ide.cpp

```diff
@@ -2545,6 +2545,8 @@ struct MCIdeFilterControlsVisitor: public MCObjectVisitor
 
             }
             break;
+            default:
+                break;
```

Comment:
I'm pretty sure this is a case where the switch should explicitly handle _all_ possible members of the enumeration.